lib/websocket/websocket-handler.ts

```diff
@@ -152,7 +158,7 @@ function setup(socket: Interface.ISocket): void
                         activeSubscriptions.set(s.correlationId, s);
                     });
                     socket.log.debug('Subscribed');
-                    socket.notifySubscribed();
+                    socket.notifySubscribed(getCorrelationIds(subscriptions));
```

Comment:
I often hesitate myself between reusing an existing loop and creating a new one. In this case I find it clearer to factor the correlation ids extraction into its own function.